tests/series_only/drop_nulls_test.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any
+
+import narwhals as nw
+from tests.utils import compare_dicts
+
+
+def test_drop_nulls(constructor: Any) -> None:
```

Comment:
can you move this over to `tests/expr_and_series/drop_nulls_test`? because you're testing `Expr.drop_nulls` here